src/harness/testcases/WINNF_FT_S_REG_testcase.py

```diff
@@ -424,6 +424,196 @@ def test_WINNF_FT_S_REG_5(self):
+    devices_with_administrative_data = [device_1, device_2, device_7, device_8,\
+                                        device_9, device_15]
+    # Inject Fcc Id and User Id for the devices that contain all Fcc and user data data
+    for device in devices_with_administrative_data:
+        self._sas_admin.InjectFccId({'fccId': device['fccId']})
+        self._sas_admin.InjectUserId({'userId': device['userId']})
+    # Inject Fcc Id with not default Fcc max Eirp
+    self._sas_admin.InjectFccId({'fccId':'fccId_approved_eirp',\
+                                  'fccMaxEirp': 25})   
+    
+    # (Generate CPI RSA keys and) Load CPI user info
+    cpi_id = 'professional_installer_id_1'
+    cpi_name = 'a_name'
+    cpi_private_key, cpi_public_key = generateCpiRsaKeys()
+    self._sas_admin.InjectCpiUser({
+        'cpiId': cpi_id,
+        'cpiName': cpi_name,
+        'cpiPublicKey': cpi_public_key
+    })
+    # Pre-load conditionals
+    conditionals_10 = {
+        'cbsdCategory': device_10['cbsdCategory'],
+        'fccId': device_10['fccId'],
+        'cbsdSerialNumber': device_10['cbsdSerialNumber'],
+        'airInterface': device_10['airInterface'],
+        'installationParam': device_10['installationParam'],
+        'measCapability': device_10['measCapability']
+    }
+    
+    conditionals_12 = {
+        'cbsdCategory': device_12['cbsdCategory'],
+        'fccId': device_12['fccId'],
+        'cbsdSerialNumber': device_12['cbsdSerialNumber'],
+        'airInterface': device_12['airInterface'],
+        'installationParam': device_12['installationParam'],
+        'measCapability': device_12['measCapability']
+    }
+    conditionals_13 = {
+        'cbsdCategory': device_13['cbsdCategory'],
+        'fccId': device_13['fccId'],
+        'cbsdSerialNumber': device_13['cbsdSerialNumber'],
+        'airInterface': device_13['airInterface'],
+        'installationParam': device_13['installationParam'],
+        'measCapability': device_13['measCapability']
+    }
+    conditionals_14 = {
+        'cbsdCategory': device_14['cbsdCategory'],
+        'fccId': device_14['fccId'],
+        'cbsdSerialNumber': device_14['cbsdSerialNumber'],
+        'airInterface': device_14['airInterface'],
+        'installationParam': device_14['installationParam'],
+        'measCapability': device_14['measCapability']
+    }
+
+    conditionals = {
+        'registrationData': [
+            conditionals_10, conditionals_12, conditionals_13, conditionals_14
+        ]
+    }
+    self._sas_admin.PreloadRegistrationData(conditionals)
+
+    # Remove conditionals from registration
+    del device_10['cbsdCategory']
+    del device_10['airInterface']
+    del device_10['installationParam']
+    del device_10['measCapability']
+    del device_12['cbsdCategory']
+    del device_12['airInterface']
+    del device_12['measCapability']
+    del device_13['cbsdCategory']
+    del device_13['airInterface']
+    del device_13['measCapability']
```

Comment:
The airInterface and cbsdCategory need to be provided outside of the CPI signed data if CPI signed data is included. So, these fields are not to be deleted here.

Reply:
Thank you for this comment, you are right.
Actually we should preload only the isntallation parameters for the devices 12, 13 ,14, not all REG Conditional data.
I updated the code.